README.md

```diff
@@ -0,0 +1,14 @@
+## Dodgeball
+Game Written in [React Native](https://reactnative.dev/) using [Expo](https://expo.dev/) and [React Native Game Engine](https://github.com/bberak/react-native-game-engine)
+
+### Quick Start
+To run these commands, you may need to install [git](https://github.com/git-guides/install-git), [npm,](https://www.npmjs.com/package/npm) and [nodejs](https://nodejs.org/en/).
+
+#### 1. Clone the repository:
+git clone https://github.com/nolan-3/Dodgeball
```

Comment:
In markdown, if you put your code inside triple backticks to format it like code:
````
```
git clone https://github.com/nolan-3/Dodgeball
```
````

Renders as:
```
git clone https://github.com/nolan-3/Dodgeball
```

Don't think too hard about how I wrote this comment.